src/dataflow/src/render/sinks.rs

```diff
@@ -40,6 +41,7 @@ where
         &mut self,
         render_state: &mut RenderState,
         tokens: &mut RelevantTokens,
+        scope: &mut Child<'g, G, G::Timestamp>,
```

Comment:
It looks like the method only needs `scope.peers()` and `scope.index()`. Probably better to pass them in for clarity.